src/_data/main-nav.yml

```diff
@@ -195,8 +195,8 @@
     - label: Checkout
       url: /howdoi/checkout/checkout_overview.html
 
-    - label: Compliance
-      url: /compliance/industry-compliance.html
+    - label: Security and Compliance
```

Comment:
These TOC entries are listed in alphabetical order. Since this is security, I wouldn't object to listing it first. Otherwise, move it down list.

Reply:
Yes, that was my thinking. So much of Compliance and Privacy hinges on Security. My take is that it should be first.

Reply:
I tried it both ways, and alphabetical order is much better.